--- CAMD/Demo/Makefile.orig	2012-05-27 01:23:01.000000000 +0200
+++ CAMD/Demo/Makefile	2012-06-27 16:48:20.148813310 +0200
@@ -4,16 +4,15 @@
 
 default: camd_simple camd_demo camd_demo2 camd_l_demo
 
-include ../../SuiteSparse_config/SuiteSparse_config.mk
+include /usr/share/misc/SuiteSparse_config.mk
 
-C = $(CC) $(CF) -I../Include -I../../SuiteSparse_config
+C = $(CC) $(CF) -I../Include
 
-INC = ../Include/camd.h ../../SuiteSparse_config/SuiteSparse_config.h
+INC = ../Include/camd.h
 
-LIB2 = ../../SuiteSparse_config/libsuitesparseconfig.a $(LIB)
+LIB2 = -lsuitesparseconfig $(LIB)
 
 library:
-	( cd ../../SuiteSparse_config ; $(MAKE) )
 	( cd ../Lib ; $(MAKE) )
 
 #------------------------------------------------------------------------------
diff -ur CAMD/Doc/Makefile CAMD-uf/Doc/Makefile
--- CAMD/Doc/Makefile	2006-04-25 18:34:27.000000000 +0200
+++ CAMD-uf/Doc/Makefile	2007-12-02 18:01:59.000000000 +0100
@@ -4,7 +4,7 @@
 
 default: dist
 
-include ../../SuiteSparse_config/SuiteSparse_config.mk
+include /usr/share/misc/SuiteSparse_config.mk
 
 #------------------------------------------------------------------------------
 # Remove all but the files in the original distribution
--- CAMD/Include/camd.h.orig	2012-05-29 18:43:27.000000000 +0200
+++ CAMD/Include/camd.h	2012-06-27 16:22:56.608877247 +0200
@@ -31,7 +31,7 @@
 /* get the definition of size_t: */
 #include <stddef.h>
 
-#include "SuiteSparse_config.h"
+#include <SuiteSparse_config.h>
 
 int camd_order              /* returns CAMD_OK, CAMD_OK_BUT_JUMBLED,
                              * CAMD_INVALID, or CAMD_OUT_OF_MEMORY */
--- CAMD/Lib/Makefile.orig	2011-12-08 18:36:42.000000000 +0100
+++ CAMD/Lib/Makefile	2012-01-01 13:52:49.706230628 +0100
@@ -8,9 +8,9 @@
 
 default: everything
 
-include ../../SuiteSparse_config/SuiteSparse_config.mk
+include /usr/share/misc/SuiteSparse_config.mk
 
-C = $(CC) $(CF) -I../Include -I../../SuiteSparse_config
+C = $(CC) $(CF) -I../Include
 
 everything:
 	$(C) -DDINT -c ../Source/camd_aat.c -o camd_i_aat.o
diff -ur CAMD/Makefile CAMD-uf/Makefile
--- CAMD/Makefile	2007-05-05 19:23:47.000000000 +0200
+++ CAMD-uf/Makefile	2007-12-02 18:02:33.000000000 +0100
@@ -4,7 +4,7 @@
 
 default: demo
 
-include ../SuiteSparse_config/SuiteSparse_config.mk
+include /usr/share/misc/SuiteSparse_config.mk
 
 # Compile all C code, including the C-callable routines.
 # Do not compile the FORTRAN versions, or MATLAB interface.
--- CAMD/Lib/GNUmakefile.orig	2012-05-27 01:26:47.000000000 +0200
+++ CAMD/Lib/GNUmakefile	2012-06-27 16:48:35.802145990 +0200
@@ -4,9 +4,9 @@
 
 default: libcamd.a
 
-include ../../SuiteSparse_config/SuiteSparse_config.mk
+include /usr/share/misc/SuiteSparse_config.mk
 
-C = $(CC) $(CF) -I../Include -I../../SuiteSparse_config
+C = $(CC) $(CF) -I../Include
 
 #-------------------------------------------------------------------------------
 # source files
@@ -15,8 +15,7 @@
 CAMD = camd_aat camd_1 camd_2 camd_dump camd_postorder camd_defaults \
 	camd_order camd_control camd_info camd_valid camd_preprocess
 
-INC = ../Include/camd.h ../Include/camd_internal.h \
-      ../../SuiteSparse_config/SuiteSparse_config.h
+INC = ../Include/camd.h ../Include/camd_internal.h
 
 #-------------------------------------------------------------------------------
 # object files for each version
